src/code/translations.json

```diff
@@ -104,7 +104,7 @@
     "ASSIGN MARKER PROMPT": "Assign marker from: ${value}",
     "ANCHOR ASSIGNMENT": " all outbound links",
     "ASSIGN OUTBOUND PROMPT": "Assign all outbound links from: ${value}",
-    "AUTH INCOMPAT": "You have activated a plugin that is incompatable with Wasabee",
+    "AUTH INCOMPAT": "You have activated a plugin in TamperMonkey that is incompatable with Wasabee",
```

Comment:
I think this is unnecessary, since TamperMonkey isn't the only one. GreaseMonkey, et.al. also exist.